internal/cmd/versions_cmd_test.go

```diff
@@ -35,72 +36,171 @@ func TestGetEnvoyVersions_NothingYet(t *testing.T) {
 	err := c.Run([]string{"getenvoy", "versions"})
 
 	require.NoError(t, err)
-	require.Equal(t, "No Envoy versions, yet\n", stdout.String())
+	require.Empty(t, stdout) // allows consistent parsing even when nothing yet installed
 	require.Empty(t, stderr)
 }
 
-func TestGetEnvoyVersions_Sorted(t *testing.T) {
-	o, cleanup := setupTest(t)
+func TestGetEnvoyVersions_NoCurrentVersion(t *testing.T) {
+	o, cleanup := setupTestVersions(t)
 	defer cleanup()
+	require.NoError(t, os.Remove(filepath.Join(o.HomeDir, "version")))
 
-	oneOneTwo := filepath.Join(o.HomeDir, "versions", "1.1.2")
-	require.NoError(t, os.MkdirAll(oneOneTwo, 0700))
-	morerequire.RequireSetMtime(t, oneOneTwo, "2020-12-31")
+	c, stdout, stderr := newApp(o)
+	o.Out = stdout
+	err := c.Run([]string{"getenvoy", "versions"})
 
-	oneTwoOne := filepath.Join(o.HomeDir, "versions", "1.2.1")
-	require.NoError(t, os.MkdirAll(oneTwoOne, 0700))
-	morerequire.RequireSetMtime(t, oneTwoOne, "2020-12-30")
+	require.NoError(t, err)
+	require.Equal(t, `  1.2.2 2021-01-31
+  1.1.2 2021-01-31
+  1.2.1 2021-01-30
+`, stdout.String())
+	require.Empty(t, stderr)
+}
 
-	oneTwoTwo := filepath.Join(o.HomeDir, "versions", "1.2.2")
-	require.NoError(t, os.MkdirAll(oneTwoTwo, 0700))
-	morerequire.RequireSetMtime(t, oneTwoTwo, "2020-12-31")
+func TestGetEnvoyVersions_CurrentVersion(t *testing.T) {
+	o, cleanup := setupTestVersions(t)
+	defer cleanup()
+
+	// These tests depend on prior state, so execute sequentially (don't change the order).
+	// The different version values are intentional to ensure overrides work properly.
+	for _, tt := range []struct {
+		name, expected string
+		setup          func() func()
+	}{
+		{"no current version", `  1.2.2 2021-01-31
```

Comment:
Could you explicitly give the field name for cases? Now it is a bit hard to read..

```suggestion
		{name: "no current version", expected:`  1.2.2 2021-01-31
```

Reply:
I can't read the tests anymore especially as go makes the indent nuts. I'm reverting matrixifying this test